release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2740,6 +2740,53 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+[id="ocp-4-15-31_{context}"]
+=== RHSA-2024:6409 - {product-title} 4.15.31 bug fix and security update
+
+Issued: 11 September 2024
+
+{product-title} release 4.15.31, which includes security updates, is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2024:6409[RHSA-2024:6409] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHBA-2024:6414[RHBA-2024:6414] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.31 --pullspecs
+----
+
+[id="ocp-4-15-31-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the `spec.noProxy` field from the cluster-wide proxy was not considered when the {cmo-first} configured proxy capabilities for a Prometheus remote write endpoint. With this release, the {cmo-short} no longer configures proxy capabilities for any remote write endpoints whose URL would bypass the proxy according to the `noProxy` field. (link:https://issues.redhat.com/browse/OCPBUGS-39172[*OCPBUGS-39172*])
+
+* Previously, utlization cards displayed limit in a way that incorrectly implied a relationship between capacity and limits. With this release, the position of limit is changed to remove this implication. (link:https://issues.redhat.com/browse/OCPBUGS-39085[*OCPBUGS-39085*])
+
+* Previously, the `openvswitch` service used older cluster configurations after a cluster upgrade and this caused the `openvswitch` service to stop. With this release, the `openvswitch` service is now restarted after a cluster upgrade so that the service uses the newer cluster configurations. (link:https://issues.redhat.com/browse/OCPBUGS-34842[*OCPBUGS-34842*])
+
+* Previously, after you submitted the same value into the {vmw-first} configuration dialog, cluster nodes unintentionally rebooted. With this release, nodes reboot after you enter new values into the dialog and not the same values.(link:https://issues.redhat.com/browse/OCPBUGS-33938[*OCPBUGS-33938*])
+
```

Comment:
ISG and SSG suggest using the term fail instead of crash in most instances. 

https://redhat-documentation.github.io/supplementary-style-guide/#crash

https://www.ibm.com/docs/en/ibm-style?topic=word-usage#c

Could the following entry be rephrased to something like the following?

 Previously, if a virtual machine (VM) was deleted and the network interface controller (NIC) still existed for that VM, the {azure-first} VM verification check failed. With this release, the verification check can now handle this situation by gracefully processing the issue without failing.

Instead of fail, another option might be "terminates unexpectedly"